composer.json

```diff
@@ -206,7 +206,7 @@
         "drupal/adminimal_theme": "1.3.0",
         "drupal/asset_injector": "2.4.0",
         "drupal/better_formats": "1.x-dev",
-        "drupal/block_class": "1.0.0-alpha1",
+        "drupal/block_class": "1.0.0",
```

Comment:
I saw this was removed and added back. Were tests failing when you removed it entirely? That may just be due to modules declaring it as a dependency?